lib/strategy/FragmentationStrategyProbabilityQuadHandling.ts

```diff
@@ -0,0 +1,99 @@
+import type * as RDF from '@rdfjs/types';
+import prand from 'pure-rand';
```

Comment:
Using a separate package for something as simple as random generation seems overkill.

Reply:
I need to handle random seeds for reproducibility and I think the native library does that.

Reply:
Ok, that's fair.